spec/eof.md

```diff
@@ -294,8 +294,10 @@ The following instructions are introduced in EOF code:
 - `RETURNDATALOAD (0xf7)` instruction
     - deduct 3 gas
     - pop `offset` from the stack
-    - if `offset + 32 > len(returndata buffer)`, execution results in an exceptional halt
     - push 1 item onto the stack, the 32-byte word read from the returndata buffer starting at `offset`
+    - if `offset + 32 > len(returndata buffer)` the result is zero-padded (same behavior as `CALLDATALOAD`).
+- `RETURNDATACOPY (0x3E)` instruction
```

Comment:
should be moved to "Modified Behavior" section, then also can to rephrase from going like "same behavior as legacy..." to just spelling the change out.

To be discussed on next EOF impl call